### ISSUE TITLE
Install ovs on centos failed due to lack of package volumedriver-no-dedup-server

### ISSUE THREAD

**Issue by winglq** [openvstorage/framework#1183]
I followed this link to install ovs on centos but it turns to the following package missing error. I see volumedriver-server is install but no volumedriver-no-dedup-server. How could I get volumedriver-no-dedup-server package? Thanks.

https://openvstorage.gitbooks.io/administration/content/Installation/centos.html

Error: Package: openvstorage-core-2.7.5_fargo.3-1.x86_64 (openvstorage)
           Requires: volumedriver-no-dedup-server >= 6.3

**Comment by wimpers** [comment]
Hi, the volumedriver-no-dedup-server 6.4.0 rpms are uploaded to unstable. Can you get them from there?

**Comment by winglq** [comment]
Thanks for the quick response. I got the rpm but met a new issue after running ovs setup. Open a new issue
